sdk/python/tests/compiler/testdata/basic_no_decorator.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Google LLC
```

Comment:
I know we appropriated those tests from Google, but since we're shameless, we should change it to 
`# Copyright 2020 kubeflow.org`